EIP-7777-SuperFuse.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7777
+title: SuperFuse — Atomic Liquidity Booster
```

Comment:
```suggestion
title: Atomic Liquidity Booster
```
SuperFuse sounds like a brand/product name.  ERCs are not for marketing/promotion so please don't include brand/product names.  The title should be descriptive, e.g. Atomic liquidity operations.

Note: I am not an ERC editor.